server/compaction-coordinator/src/main/java/org/apache/accumulo/coordinator/CompactionCoordinator.java

```diff
@@ -653,6 +679,21 @@ protected void startFailureSummaryLogging() {
     ThreadPools.watchNonCriticalScheduledTask(future);
   }
 
+  private <T> void printFailures(String logPrefix,
+      ConcurrentHashMap<T,FailureCounts> failureCounts) {
+    for (var key : failureCounts.keySet()) {
+      failureCounts.compute(key, (k, counts) -> {
+        if (counts != null && counts.failures > 0) {
+          LOG.warn("{} {} failures:{} successes:{} since last time this was logged ", logPrefix, k,
```

Comment:
I don't think we are removing from the map anymore, so I'm not sure that these counts are relative to the last time they are logged. I think they are since the start of the process. Also, since we are not removing from the map, old tables, queues, and compactors will still be reported.